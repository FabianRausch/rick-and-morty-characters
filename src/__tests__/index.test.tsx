import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import Component from "../pages/error";

describe("Test on Error page", () => {
  it("Should match with snapshot", () => {
    const { asFragment } = render(<Component />);
    expect(asFragment).toMatchSnapshot();
  });
});
