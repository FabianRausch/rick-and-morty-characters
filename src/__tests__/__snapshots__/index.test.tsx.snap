// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Test on Error page Should match with snapshot 1`] = `[Function]`;
